drivers/display/ssd1306.c

```diff
@@ -371,6 +406,34 @@ static int ssd1306_init(const struct device *dev)
 			   DT_INST_GPIO_FLAGS(0, reset_gpios));
 #endif
 
+#if DT_INST_ON_BUS(0, spi)
+	driver->spi_config.frequency = 10000000;
```

Comment:
driver->spi_config.frequency = DT_INST_PROP(0, spi_max_frequency);

Reply:
Is corrected now.